### DISCUSSION TITLE
Gradient vanishing of group attention during pre-training

### DISCUSSION THREAD

**Discussion by gwyanCN** [amazon-science/chronos-forecasting#487] [Q&A]
Hello, we are currently trying to implement our own multivariate Chronos model. During pre-training on both the synthetic and real-world data described in the paper, we found that as training progresses, the gradient ratio between time attention and group attention becomes increasingly imbalanced (fig 1), and group attention gradually suffers from gradient vanishing. Furthermore, during inference testing, we observed that group attention has no effect (fig 2). Have you encountered this issue before, and if so, how did you resolve it?

fig1:
<img width="1174" height="582" alt="output" src="https://github.com/user-attachments/assets/b00847c6-9c01-49dc-9d13-7b5d732b63bc" />

fig2:
the attention weights of group attention in the first layer and the tenth layer.
Above is ours, below is the open-source.
<img width="948" height="780" alt="image" src="https://github.com/user-attachments/assets/028ee2f3-94f9-48f2-a355-5c42cd2d1186" />
